parse.c

```diff
@@ -39,7 +39,7 @@ static void print_radiotap_namespace(struct ieee80211_radiotap_iterator *iter)
 {
 	switch (iter->this_arg_index) {
 	case IEEE80211_RADIOTAP_TSFT:
-		printf("\tTSFT: %llu\n", le64toh(*(unsigned long long *)iter->this_arg));
+		printf("\tTSFT: %lu\n", le64toh(*(unsigned long long *)iter->this_arg));
```

Comment:
you probably need a cast on the value, otherwise you're just moving the warning around between 32/64 bit